haiku-libs/sagebrush/sagebrush-1.0.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="An interface library for image editor"
+DESCRIPTION="SageBrush is an interface library for image editor. It provides \
+a kit to build an image editor easily without starting from scratch."
+HOMEPAGE="https://github.com/HaikuArchives/SageBrush"
+COPYRIGHT="2012 Vladislav Burundukov"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="d9b3945cb78c7070aa8b48d2e0bc6a2007fa3601"
+SOURCE_URI="https://github.com/HaikuArchives/SageBrush/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d152e0d572a8eb0b8fc39a531cc1466827a9b96526bc5669dd181ef77ed32940"
+SOURCE_FILENAME="SageBrush-$portVersion-$srcGitRev.tar.gz"
```

Comment:
you can remove the portversion here

Reply:
No, we should not remove the `-$portVersion` here.
Just adding a comment here so that we understand why the `-$portVersion` is back.
See [this comment](https://github.com/haikuports/haikuports/pull/2099#discussion_r160833040) from @alaviss.

BTW, as @alaviss also suggests in the same comment, we should also rename the recipe to: `sagebrush-1.0.0~git.recipe`.
Well, I'd say that if we don't have any release yet, maybe we could even replace the "1.0.0" by a single "0", thus renaming the recipe to `sagebrush-0~git.recipe`.
@alaviss (and others), what do you think?